nvflare/ha/ha_admin_cmds.py

```diff
@@ -15,78 +15,20 @@
 from nvflare.fuel.hci.reg import CommandModule, CommandModuleSpec, CommandSpec
 from nvflare.security.logging import secure_format_exception
 
 
-def list_sp(args, ctx: CommandContext):
-    """Lists service provider information based on the last heartbeat from the overseer."""
-    overseer_agent = ctx.get_api().service_finder.overseer_agent
-    return {
-        "status": APIStatus.SUCCESS,
-        "details": str(overseer_agent.overseer_info),
-        "data": overseer_agent.overseer_info,
-    }
-
-
-def get_active_sp(args, ctx: CommandContext):
-    overseer_agent = ctx.get_api().service_finder.overseer_agent
-    return {"status": APIStatus.SUCCESS, "details": str(overseer_agent.get_primary_sp())}
-
-
-def promote_sp(args, ctx: CommandContext):
-    overseer_agent = ctx.get_api().service_finder.overseer_agent
-    if len(args) != 2:
-        return {"status": APIStatus.ERROR_SYNTAX, "details": "usage: promote_sp example1.com:8002:8003"}
-
-    sp_end_point = args[1]
-    resp = overseer_agent.promote_sp(sp_end_point)
-    if json.loads(resp.text).get("Error"):
-        return {
-            "status": APIStatus.ERROR_RUNTIME,
-            "details": "Error: {}".format(json.loads(resp.text).get("Error")),
-        }
-    else:
-        return {
-            "status": APIStatus.SUCCESS,
-            "details": "Promoted endpoint: {}. Synchronizing with overseer...".format(sp_end_point),
-        }
-
-
-def shutdown_system(args, ctx: CommandContext):
-    api = ctx.get_api()
-    overseer_agent = api.service_finder.overseer_agent
-    try:
-        status = api.do_command("check_status server").get("data")
-        if status[0].get("data") != "Engine status: stopped":
-            return {
-                "status": APIStatus.ERROR_RUNTIME,
-                "details": "Error: There are still jobs running. Please let them finish or abort_job before attempting shutdown.",
-            }
-    except Exception as e:
-        return {
-            "status": APIStatus.ERROR_RUNTIME,
-            "details": f"Error getting server status to make sure all jobs are stopped before shutting down system: {secure_format_exception(e)}",
-        }
-    print("Shutting down the system...")
-    resp = overseer_agent.set_state("shutdown")
-    if json.loads(resp.text).get("Error"):
-        return {
-            "status": APIStatus.ERROR_RUNTIME,
-            "details": "Error: {}".format(json.loads(resp.text).get("Error")),
-        }
-    else:
-        return {"status": APIStatus.SUCCESS, "details": "Set state to shutdown in overseer."}
-
-
 class HACommandModule(CommandModule):
     """Command module with commands for management in relation to the high availability framework."""
 
     def __init__(self, overseer_agent: OverseerAgent):
-        self.overseer_agent = overseer_agent
+        self.overseer_agent = overseer_agent  # we should not need self.overseer_agent anymore
         self.logger = logging.getLogger(self.__class__.__name__)
```

Comment:
just a note, can be done in another PR later, these 2 things are not used we can remove them both.